.travis.yml

```diff
@@ -99,19 +100,21 @@ install:
   pip install git+https://github.com/swryan/coveralls-python@work;
 
   echo " >> Installing pyOptSparse";
-  echo "  > Cloning pyOptSparse from mdolab";
-  git clone https://github.com/mdolab/pyoptsparse.git;
-  cd pyoptsparse;
-  git checkout tags/v1.2;
-
-  if [ "$SNOPT_LOCATION" ] && [ "${PY:0:1}" = "3" ]; then
-    cd pyoptsparse/pySNOPT;
+  git clone https://gist.github.com/b34594cc2a4468e9d215cef2f4187b71.git pyopt_ipopt_install;
+  mv pyopt_ipopt_install/* .
+  rm -rf pyopt_ipopt_install
+  chmod 755 ./build_pyoptsparse_ipopt.sh; ./build_pyoptsparse_ipopt.sh;
+
+  if [ "$SNOPT_LOCATION72" ] && [ "${PY:0:1}" = "3" ]; then
+    echo " >> Installing SNOPT";
+    cd pyoptsparse/pyoptsparse/pySNOPT;
     echo "  > Secure copying SNOPT over SSH";
-    scp -r "$SNOPT_LOCATION" .;
-    cd ../..;
+    scp -r "$SNOPT_LOCATION72" .;
+    cd ../../..;
```

Comment:
Looking at the skipped tests, it appears that SNOPT is not being built.
See https://github.com/DKilkenny/OpenMDAO/pull/9 for a fix and slightly clean up logic.

Reply:
Also updated to use build script from new OpenMDAO repo: 
https://github.com/OpenMDAO/build_pyoptsparse